packages/puppeteer-core/src/bidi/Frame.ts

```diff
@@ -301,10 +302,18 @@ export class BidiFrame extends Frame {
       // readiness=interactive.
       //
       // Related: https://bugzilla.mozilla.org/show_bug.cgi?id=1846601
-      this.browsingContext.navigate(
-        url,
-        Bidi.BrowsingContext.ReadinessState.Interactive
-      ),
+      this.browsingContext
+        .navigate(url, Bidi.BrowsingContext.ReadinessState.Interactive)
+        .catch(error => {
+          if (
+            isErrorLike(error) &&
+            error.message.includes('net::ERR_HTTP_RESPONSE_CODE_FAILURE')
+          ) {
+            return;
```

Comment:
can we return the network response here?

Reply:
We do that is handled by the `this.waitForNavigation(options),` not by this command. this needs to not throw.